### ISSUE TITLE
[Bugbash][H2][T12][Healtheintent][Functional] Registries: Population Heading is also seelctable and provides data.It doesnt look like a header of organisation list.

### ISSUE THREAD

**Issue by pavithrasherigar** [MohanRao13/BugBash_December#345]
[Bugbasg][H2][T12][Healtheintent][Functional] Registries: Population Heading is also seelctable and provides data.It doesnt look like a header of organisation list.

![image](https://cloud.githubusercontent.com/assets/24434364/21045731/86839c68-be27-11e6-965d-8bb2077681d8.png)

**Comment by alexjose74** [comment]
This is correct. The population header gives data pertaining to all the people under all the organizations. This is not a header.
